Python/Product/PythonTools/missing_req_packages.py

```diff
@@ -0,0 +1,106 @@
+# Python Tools for Visual Studio
+# Copyright(c) Microsoft Corporation
+# All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the License); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at http://www.apache.org/licenses/LICENSE-2.0
+# 
+# THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+# IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+# MERCHANTABILITY OR NON-INFRINGEMENT.
+# 
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+
+#This script determines if a python interpreter is missing any packages 
+#As defined in a text file (usually requirements.txt) which goes into pip
+
+#Exit code = 1 means atleast 1 package is missing or version conflict. Show install package bar
+#Exit code = 0 means that packages may or may not be missing or file not find or other problems. Do not show install bar
```

Comment:
Same comment here as I've put below for C# comments, please leave a space between # and the text